### ISSUE TITLE
Name LanguageModelChat is confusing for me.

### ISSUE THREAD

**Issue by dbaeumer** [microsoft/vscode#213632]
Testing #213514

From reading a documentation it is a language model to do chat request. Would it make more sense to name it `ChatLanguageModel`. The name `LanguageModelChat` for me point to a chat in a language model.

**Comment by jrieken** [comment]
I do agree that it needs some "getting used to" the reasoning is to scope the type by names, e.g `LanguageModelXYZ` because I see a near future in which we also have `LanguageModelEmbeddings`. This gives type names some namespacing even though it appear the wrong way around

**Comment by roblourens** [comment]
I agree that it seems odd at first glance but I think it's fine, and when it sits next to `LanguageModelEmbeddings` I think it will look better.

**Comment by jrieken** [comment]
closing this without change
